packages/server/src/api/rest/index.ts

```diff
@@ -618,14 +634,24 @@ class RequestHandler {
                     paginator: this.makePaginator(url, offset, limit, total),
                 },
             };
+            let body = await this.serializeItems(type, entities, options);
+            body = this.addTotalCountMeta(body, total);
 
             return {
                 status: 200,
-                body: await this.serializeItems(type, entities, options),
+                body: body,
             };
         }
     }
 
+    private addTotalCountMeta(body: Partial<DataDocument<any>>, total: number | undefined): Partial<DataDocument<any>> {
+        if (total) {
```

Comment:
Shall we explicitly chek `undefined` so that `0` count can be included?

Reply:
Also, shall we merge `total` into `body.meta` instead of overwriting it? Just thinking maybe in the future there'll already be a `meta` object in body containing other stuff.